src/Service/VendorService/EReolenGlobal/EReolenGlobalVendorService.php

```diff
@@ -0,0 +1,207 @@
+<?php
+/**
+ * @file
+ * Service for updating data from 'eReolen Global / overdrive.com''.
+ */
+
+namespace App\Service\VendorService\EReolenGlobal;
+
+use App\Exception\IllegalVendorServiceException;
+use App\Exception\UnknownVendorServiceException;
+use App\Service\VendorService\AbstractBaseVendorService;
+use App\Service\VendorService\ProgressBarTrait;
+use App\Utils\Message\VendorImportResultMessage;
+use App\Utils\Types\IdentifierType;
+use Doctrine\ORM\EntityManagerInterface;
+use GuzzleHttp\ClientInterface;
+use GuzzleHttp\Exception\GuzzleException;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+
+/**
+ * Class EReolenGlobalVendorService.
+ */
+class EReolenGlobalVendorService extends AbstractBaseVendorService
+{
+    use ProgressBarTrait;
+
+    protected const VENDOR_ID = 14;
+
+    private const AUTH_URL = 'https://oauth.overdrive.com/token';
+    private const AUTH_GRANT_TYPE = 'grant_type=client_credentials';
+
+    private const ACCOUNT_API_URL = 'https://api.overdrive.com/v1/libraries/6247';
+
```

Comment:
Should some of these URL have been placed in the vendor in the database to ensure that we chould change them without releasing an new version?

Database have: `data_server_uri` and `image_server_uri` used by bogportalen!

Reply:
These are versioned API URLs. They can only change if this version of the API is no longer available. 

Additionally the DB schema doesn't map well to this vendor. 

In general I believe we need to refactor the DB schema to allow for each vendor to save a custom config object to the db (json column?) with the options the vendor needs. However this is out of scope for this PR